db/migrate/20160824210800_create_students.rb

```diff
@@ -0,0 +1,17 @@
+class CreateStudents < ActiveRecord::Migration[5.0]
+  def change
+    create_table :students do |t|
+      t.string   :first_name, null: false
+      t.string   :last_name, null: false
+      t.string   :grade_level, null: false
+      t.string   :gender, null: false
+      t.string   :gpa, null: false
+      t.string   :detention_count, default: 0
```

Comment:
There's a type mismatch here. We'd prefer to make the column an integer column so it can be counted (total detentions per teacher team or something?).